site/e2e/tests/deployment/idpOrgSync.spec.ts

```diff
@@ -117,7 +117,7 @@ test.describe("IdpOrgSyncPage", () => {
 		requiresLicense();
 		await setupApiCalls(page);
 
-		await createOrganizationWithName("developers");
+		await createOrganizationWithName("admins");
```

Comment:
```suggestion
		await createOrganizationWithName(randomName());
```

avoid fixed names in e2e tests